Add aNotepad

prem-k-r opened the pull request:
![image](https://github.com/user-attachments/assets/661f27b9-b870-4d34-99df-f7f0108be354)

**Issue:** closes -

**Popularity metric:**

[Similarweb](https://www.similarweb.com/website/anotepad.com/#overview):

![image](https://github.com/user-attachments/assets/30184a5a-c81d-4f39-8a10-85942b6a81f8)


### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Took primary-color from source website for hex

prem-k-r committed:
Add aNotepad

PeterShaggyNoble reviewed (approved):
All looks perfect, @prem-k-r :+1:

Although the source file includes a few [excess points](https://github.com/user-attachments/assets/0a3cd486-6b32-40f7-9e04-6604fd0bf371) in the pencil that we might be able to remove. Could you give that a try before we merge, please?

prem-k-r committed:
Remove some nodes

adamrusted reviewed (changes requested):
Might need to go with a higher precision on this one @prem-k-r - see your icon in black and the source icon in red, there are some differences to the curves.

![image](https://github.com/user-attachments/assets/d9978743-859d-4171-abe7-e9fd497c103e)

prem-k-r committed:
Update anotepad.svg

prem-k-r commented:
> Might need to go with a higher precision on this one @prem-k-r - see your icon in black and the source icon in red, there are some differences to the curves.

ah, I earlier deleted two nodes here, fixed now


>> Although the source file includes a few [excess points](https://github.com/user-attachments/assets/0a3cd486-6b32-40f7-9e04-6604fd0bf371) in the pencil that we might be able to remove. Could you give that a try before we merge, please?

![image](https://github.com/user-attachments/assets/988f670b-4b1d-430f-b1db-54ea61144169)
![image](https://github.com/user-attachments/assets/57230972-9415-4d8d-93e3-e201033c640c)

I have deleted these nodes and fixed the rought curve; but it now differ from source, is it allowed?
![image](https://github.com/user-attachments/assets/20c91d23-e542-4951-bfae-ee7c27349a97)

adamrusted reviewed (changes requested):
Hey @prem-k-r - thanks for fixing the merge errors on this, though I'm still seeing some differences between the PR (black) and source file (red) - could you have a look for me please?

![image](https://github.com/user-attachments/assets/20775e92-91e2-4d34-bf05-753e423bdcef)
![image](https://github.com/user-attachments/assets/3fbc15b6-a6d2-4100-b64d-590ea092d8be)

prem-k-r committed:
Update anotepad.svg

prem-k-r committed:
Update anotepad.svg
